Neuroimaging Special Interest Group (#2542)
nf-core/website: src/content/special-interest-groups/neuroimaging/index.md

---
title: Introduction
subtitle: 'Community of researchers using Nextflow and nf-core for Neuroimaging'
groupName: Neuroimaging
leads:
  - AlexVCaron: Alex Valcourt Caron
  - arnaudbore: Arnaud Boré

pipelines:

>   - tractoflow

Reviewer: this is not yet an nf-core pipeline is it? in that case this will generate a broken link in the sidebar I am afraid

Author: True, it's not yet, I'll remove

---

# Introduction

> The Neuroimaging SIG aims to accelerate the integration of Nextflow and nf-core in all fields of neuroimaging. Its first goal is to put in place guidelines and standards tailored for neuroimaging. Its second is to create educational content and a common base of tailored modules and subworkflows for neuroimaging purposes, in order to streamline the integration of Nextflow and nf-core pipelines in neuroimaging, and of new researchers in the nf-core community.

Reviewer: ```suggestion
The neuroimaging Special Interest Group aims to accelerate the integration of Nextflow and nf-core in all fields of neuroimaging. Its first goal is to put in place guidelines and standards tailored for neuroimaging. Its second is to create educational content and a common base of tailored modules and subworkflows for neuroimaging purposes, in order to streamline the integration of Nextflow and nf-core pipelines in neuroimaging, and of new researchers in the nf-core community.
```
Please avoid using the acronym for Specia Interest Group 🙂

> To do so, the SIG need to indentify :

Reviewer: ```suggestion
To do so, the Special Interest Group needs to identify:
```

> 1. Common use-cases and challenges of doing Neuroimaging with Nextflow and nf-core

Reviewer: ```suggestion
1. Common use-cases and challenges of doing neuroimaging with Nextflow and nf-core
```

> 2. Educational requirements to get the Neuroimaging field in line with Nextflow and nf-core

Reviewer: ```suggestion
2. Educational requirements to get the neuroimaging field in line with Nextflow and nf-core
```

> 3. Nextflow and nf-core practices that need to translate to Neuroimaging

Reviewer: ```suggestion
3. Nextflow and nf-core practices that need to translate to neuroimaging
```

> 4. Good practices in Neuroimaging that could or need to translate to Nextflow and nf-core

Reviewer: ```suggestion
4. Good practices in neuroimaging that could or need to translate to Nextflow and nf-core
```

# Activities

> Initial activities will involve recrutement of researchers from other Neuroimaging fields into the SIG, and a monthly meeting to put in place the required knowledge. Other activities will be added once a first meeting has taken place.

Reviewer: ```suggestion
Initial activities will involve recruitment of researchers from other neuroimaging fields into the Special Interest Group, and a monthly meeting to put in place the required knowledge. Other activities will be added once a first meeting has taken place.
```
